pages/features.html

```diff
@@ -112,289 +142,65 @@
-      <H2><img src="{% static '/static/images/ic_preferences_replaygain.png' %}" class="feature_icon">{% trans "ReplayGain Normalization" %}</H2>
-      <p>
-        {% trans "Song gain is automatically adjusted when loading a deck so your mixes always maintain a consistent volume. Mixxx reads existing ReplayGain tags and analyzes songs that don't have them." %}
+      <h2><img src="{% static '/static/images/ic_library_recordings.png' %}" class="feature_icon">Recording</h2>
+      <!--TODO: clarify. What does "optional" mean?-->
+      <p>{% trans "Record your live mixes to the lossless WAV or lossy Ogg Vorbis formats, with MP3 recording enabled by separate LAME library. Recordings are automatically archived in the library for fast access." %}
       </p>
     </div>
-
     <div class="halfbox_right">
-      <H2><img src="{% static '/static/images/ic_preferences_soundhardware.png' %}" class="feature_icon">{% trans "Quad Microphone Input" %}</H2>
+      <h2><img src="{% static '/static/images/ic_preferences_broadcast.png' %}" class="feature_icon">{% trans "Live Broadcasting" %}</h2>
       <p>
-        {% trans "Whether you're dropping thick reggae vocals over some dub, giving shoutouts on the air, or throwing a rap battle, microphone input is as easy as toggling one of the talkover buttons." %}
+        {% trans "Stream your mixes live over the Internet to your audience and friends through a Shoutcast or Icecast server." %}
       </p>
     </div>
 
     <div style="clear: both;"></div>
 
-    <H1>{% trans "Comprehensive DJ Hardware Support" %}</H1>
     <div class="halfbox_left">
-      <div style="height: 200px;">
-        <center><img width="340px" src="{% static '/static/images/feature_midi-2.0.png' %}" style="padding-top: 20px;"></center>
-      </div>
-
-      <H2>{% trans "DJ Controller Support" %}</H2>
+      <h2><img src="{% static '/static/images/ic_preferences_replaygain.png' %}" class="feature_icon">{% trans "ReplayGain Normalization" %}</h2>
       <p>
-        {% blocktrans with hardware_compatibility_wiki_link="/wiki/doku.php/hardware_compatibility" %}<b>Over 30 controllers supported</b><br> including the Traktor Kontrol S4 MK2, Vestax VCI-400, Hercules RMX, Numark Mixtrack, Stanton SCS.3d, and many more. <a href="{{ hardware_compatibility_wiki_link }}">Read more{% endblocktrans %}</a>
-      </p>
-
-      <p><b>{% trans "Advanced Controller Scripting Engine" %}</b><br>
-        {% blocktrans with midi_scripting_wiki_link="/wiki/doku.php/midi_scripting" %}Mixxx features a cutting-edge JavaScript-based scripting engine for maximum flexibility. MIDI or HID messages can be mapped onto custom functions which execute complex behaviour. <a href="{{ midi_scripting_wiki_link }}">Read more</a>{% endblocktrans %}
+        {% trans "Song gain is automatically adjusted when loading a deck so your mixes maintain a consistent volume. Mixxx reads existing ReplayGain tags and analyzes songs that don't have them." %}
```

Comment:
gain -> loudness  (even though that's not technically true, it's a more widely-understandable term)